Modules/Office365DSC/Modules/O365DSCReverseGUI.psm1

```diff
@@ -391,6 +391,15 @@ function Show-O365GUI
         $txtTenantAdmin.Left = 1060
         $txtTenantAdmin.Width = 175
         $txtTenantAdmin.Font = [System.Drawing.Font]::new($txtTenantAdmin.Font.Name, 10)
+        # Pre-populate the tenant admin box with the global admin username, if we already know it (e.g. from a recent attempt), otherwise try to guess it from the current user's UPN
```

Comment:
I don't think we should try to do this honestly. Chances are that user may be different on each extraction, and for sure the user will never be a windows users. I say we leave that part out from this PR

Reply:
OK, removing.